buildSrc/src/main/java/Dependencies.kt

```diff
@@ -284,11 +284,5 @@ object RepoMatching {
     const val comGoogleFirebase = "com\\.google\\.firebase"
     const val comGoogleTesting = "com\\.google\\.testing\\..*"
```

Comment:
We want to remove these two regex options as well along with their usage, then add the new `comGoogle` as a replacement for the removed ones.

Reply:
okay I got it